app/services/EventSinkService.scala

```diff
@@ -6,33 +6,21 @@ import java.net.URI
 import java.time.Instant
 import play.api.{Logger, Play}
 import play.api.Play.current
-import play.api.libs.json.{JsValue, Json}
+import play.api.libs.json.{JsObject, JsValue, Json}
 
 object EventSinkService {
   val EXCHANGE_NAME_CONFIG_KEY = "eventsink.exchangename"
   val QUEUE_NAME_CONFIG_KEY = "eventsink.queuename"
 
   val EXCHANGE_NAME_DEFAULT_VALUE = "clowder.metrics"
   val QUEUE_NAME_DEFAULT_VALUE = "event.sink"
```

Comment:
```suggestion
  val QUEUE_NAME_DEFAULT_VALUE = ""
```

Set default queue name to "" to send to entire exchange by default